abell/models/asset_type.py

```diff
@@ -112,7 +120,14 @@ def remove_type(self):
                 'DB. Remove all assets of type %s then '
                 'resubmit this call' % self.asset_type}
 
-    def update_keys(self, remove_keys=[], managed_keys=[], unmanaged_keys=[]):
+    def update_keys(self, remove_keys=None, managed_keys=None, 
+                    unmanaged_keys=None):
+        if remove_keys is None:
+            remove_keys = []
+        if managed_keys is None:
+            managed_keys = []
+        if unmanged_keys is None:
```

Comment:
https://travis-ci.org/rcbops/abell/builds/256943677#L548 - this should be `unmanaged_keys`, missing an 'a'